tests/test_outputs/test_form_output_prep.py

```diff
@@ -0,0 +1,361 @@
+"""Tests for 'form_output_prep.py'."""
+# Standard Library Imports
+import pytest
+
+# Third Party Imports
+import pandas as pd
+import numpy as np
+
+# Local Imports
+from src.outputs.form_output_prep import form_output_prep
+
+class TestFormOutputPrep(object):
+    """Tests for form_output_prep."""
+
+    @pytest.fixture(scope="function")
+    def estimated_df(self) -> pd.DataFrame:
+        """'estimated_df' input for form_output_prep.
+        
+        NOTE: This is a subset of columns for testing purposes
+        """
+        columns = [
+            "reference", 
+            "instance", 
+            "period",
+            "201",
+            "202",
+            "211",
+            "602", 
+            "604", 
+            "a_weight", 
+            "formtype",
+            "pg_numeric"
+        ]
+        data = [
+            [39900000404, 0, 202212, 'AA', np.nan, np.nan, 100.0, 'Yes',
+                6.727272727272728, 6, 40],
```

Comment:
on this and your other PR, can you keep all the data for a dataframe row on the same line? 
If the line length causes problems with the pre-commits, we need to address these.

Reply:
Done here. Will update in other PRs where needed.